server/src/test/java/net/consensys/eventeumserver/integrationtest/ArraysIT.java

```diff
@@ -39,63 +39,64 @@
-        assertEquals(ContractEventStatus.UNCONFIRMED, eventDetails.getStatus());
+                assertEquals(registeredFilter.getEventSpecification().getEventName(), eventDetails.getName());
+                assertEquals(ContractEventStatus.UNCONFIRMED, eventDetails.getStatus());
 
-        final ArrayList<NumberParameter> uintArray =
-                (ArrayList<NumberParameter>) eventDetails.getNonIndexedParameters().get(0).getValue();
+                final ArrayList<NumberParameter> uintArray = (ArrayList<NumberParameter>) eventDetails
+                                .getNonIndexedParameters().get(0).getValue();
 
-        assertEquals(BigInteger.ONE, uintArray.get(0).getValue());
-        assertEquals(BigInteger.TEN, uintArray.get(1).getValue());
+                assertEquals(BigInteger.ONE, uintArray.get(0).getValue());
+                assertEquals(BigInteger.TEN, uintArray.get(1).getValue());
 
-        final ArrayList<StringParameter> bytesArray =
-                (ArrayList<StringParameter>) eventDetails.getNonIndexedParameters().get(1).getValue();
+                final ArrayList<StringParameter> bytesArray = (ArrayList<StringParameter>) eventDetails
+                                .getNonIndexedParameters().get(1).getValue();
 
-        assertEquals(BYTES_VALUE_HEX, bytesArray.get(0).getValue());
-        assertEquals(BYTES_VALUE2_HEX, bytesArray.get(1).getValue());
+                assertEquals(BYTES_VALUE_HEX, bytesArray.get(0).getValue());
+                assertEquals(BYTES_VALUE2_HEX, bytesArray.get(1).getValue());
 
-        assertEquals(Web3jUtil.getSignature(registeredFilter.getEventSpecification()),
-                eventDetails.getEventSpecificationSignature());
-    }
+                assertEquals(Web3jUtil.getSignature(registeredFilter.getEventSpecification()),
+                                eventDetails.getEventSpecificationSignature());
+        }
 
-    private ContractEventFilter registerDummyEventArrayFilter(String contractAddress) {
-        return registerEventFilter(createDummyEventArrayFilter(contractAddress));
-    }
+        private ContractEventFilter registerDummyEventArrayFilter(String contractAddress) {
+                return registerEventFilter(createDummyEventArrayFilter(contractAddress));
+        }
 
-    private ContractEventFilter createDummyEventArrayFilter(String contractAddress) {
+        private ContractEventFilter createDummyEventArrayFilter(String contractAddress) {
 
-        final ContractEventSpecification eventSpec = new ContractEventSpecification();
+                final ContractEventSpecification eventSpec = new ContractEventSpecification();
 
-        eventSpec.setNonIndexedParameterDefinitions(
-                Arrays.asList(
-                        new ParameterDefinition(0, ParameterType.build("UINT256[]")),
-                        new ParameterDefinition(1, ParameterType.build("BYTES32[]"))));
+                eventSpec.setNonIndexedParameterDefinitions(
```

Comment:
Use java google coding stylesheet in your IDE: https://plugins.jetbrains.com/plugin/8527-google-java-format